qualang_tools/config/components.py

```diff
@@ -355,10 +355,9 @@ class Element(ConfigBuilderElement):
     def __init__(
         self,
         name: str,
-        analog_input_ports: Optional[List[AnalogOutputPort]] = None,
-        analog_output_ports: Optional[List[AnalogInputPort]] = None,
-        digital_input_ports: Optional[List[DigitalOutputPort]] = None,
-        digital_output_ports: Optional[List[DigitalInputPort]] = None,
+        element_analog_inputs: Optional[List[AnalogOutputPort]] = None,
+        element_analog_outputs: Optional[List[AnalogInputPort]] = None,
+        element_digital_inputs: Optional[List[DigitalOutputPort]] = None,
```

Comment:
You've removed the digital outputs?
They are currently not being used (as they are for the OPD), but they will be used. Up to you.

Reply:
I also see it is not in the docstring below

Reply:
We (with Tomer) decided to postpone supporting digital_outputs for now and instead focused on essential things (the other PR) to cover OPX related things. A full list of missing things are listed here: https://github.com/qua-platform/py-qua-tools/issues/47

So it made sense to remove the digital_outputs from the arguments temporarily and I left the remaining code. FYI, ports in element_digital_outputs are actually not being used and they do not appear in the final QUA configuration.